Library/Homebrew/extend/os/mac/diagnostic.rb

```diff
@@ -264,8 +264,14 @@ def check_for_latest_xquartz
 
         <<-EOS.undent
           Your XQuartz (#{installed_version}) is outdated.
-          Please install XQuartz #{latest_version} (or delete it):
-            https://xquartz.macosforge.org
+          Please install XQuartz #{latest_version} (or delete the current version).
+          XQuartz can be installed via homebrew-cask with
+
+            brew cask install xquartz
+
+          or manually, by downloading the package from
+
+            https://www.xquartz.org/
```

Comment:
Use `#{Formatter.url("https://www.xquartz.org/")}` here, and remove the empty line above this URL, and above the `brew cask` command above.